fedimint-server/src/net/api.rs

```diff
@@ -613,125 +577,11 @@ impl<T: Clone> ExpiringCache<T> {
 
 #[cfg(test)]
 mod tests {
+    use std::time::Duration;
 
-    use fedimint_core::api::ConsensusContribution;
     use fedimint_core::task;
-    use fedimint_core::time::now;
-
-    use super::*;
-    #[test]
-    fn test_server_status_all_ok() {
```

Comment:
These tests still make sense, with minor modifications

Reply:
Yes but without the flagging grace time the calculation of the ConsensusStatus is so simple now that tests seemed unnecessary and this allows me to remove the indirection of having code that is only called in one place in a seperate function for testing purposes only.

Reply:
I don't see removing a function as a big win compared to the reduced test coverage. Anyway we can add the tests back later.